elementary/monitor/data_monitoring.py

```diff
@@ -514,8 +532,11 @@ def _check_dbt_package_compatibility(dbt_pkg_ver: str):
             dbt_pkg_ver.major != py_pkg_ver.major
             or dbt_pkg_ver.minor != py_pkg_ver.minor
         ):
-            click.secho(
+            logger.warning(
                 f"You are using incompatible versions between edr ({py_pkg_ver}) and Elementary's dbt package ({dbt_pkg_ver}).\n "
                 "Please upgrade the major and minor versions to align.\n",
-                fg="yellow",
             )
+
+    def run_artifacts(self):
```

Comment:
Should run all of Elementary's models?
Will take longer but solves compatibility issues in case we added columns etc.

Reply:
I think so.
Our models are relative small and will add small amount of running time IMO.
So lets make sure we are fully updated :)